tutorials/Qubit/Qubit.ipynb

```diff
@@ -0,0 +1,309 @@
+    "    \n",
+    "    // This line allocates a qubit in state |0⟩\n",
+    "    using (q = Qubit()) {\n",
+    "        \n",
+    "        \n",
+    "        Message(\"State |0⟩:\");\n",
+    "        \n",
+    "        // This line prints out the state of the quantum computer\n",
+    "        // Since only one qubit is allocated, only its state is printed\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This line changes the qubit from state |0⟩ to state |1⟩\n",
+    "        X(q);\n",
+    "        \n",
+    "        Message(\"State |1⟩:\");\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This line changes the qubit to state |-⟩ = (1/sqrt(2))(|0⟩ - |1⟩)\n",
+    "        // That is, this puts the qubit into a superposition\n",
+    "        // 1/sqrt(2) is approximately 0.707107\n",
+    "        H(q);\n",
+    "        \n",
+    "        Message(\"State |-⟩:\");\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This line changes the qubit to state |-i⟩ = (1/sqrt(2))(|0⟩ - i|1⟩)\n",
+    "        S(q);\n",
+    "        \n",
+    "        Message(\"State |-i⟩:\");\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This will put the qubit into an uneven superposition,\n",
+    "        // where the amplitudes of |0⟩ and |1⟩ have different moduli\n",
+    "        Rx(2.0, q);\n",
+    "        \n",
+    "        Message(\"Uneven superposition state:\");\n",
+    "        DumpMachine();\n",
+    "        \n",
+    "        // This line returns the qubit to state |0⟩\n",
+    "        Reset(q);\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%simulate Qubits_Demo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Relative and Global Phase\n",
+    "\n",
+    "You may recall that a complex number has a parameter called its phase. If a complex number $x$ is written in polar form $x = re^{i\\theta}$, its phase is $\\theta$.\n",
+    "\n",
+    "The phase of a basis state is the complex phase of the amplitude of that state. For example, a system in state $\\frac{1 + i}{2}|0\\rangle + \\frac{1 - i}{2}|1\\rangle$, the phase of $|0\\rangle$ is $\\frac{\\pi}{4}$, and the phase of $|1\\rangle$ is $-\\frac{\\pi}{4}$. The difference between these two phases is known as **relative phase**.\n",
+    "\n",
+    "The state of a qubit (or, more generally, the state of a quantum system) is defined up to a global phase. That is, applying a phase to the entire state of a system (multiplying the entire vector by $e^{i\\theta}$ for any real $\\theta$) doesn't actually affect the state of the system.\n",
```

Comment:
The flow of thought seems a bit weird to me here - we postulate this before defining what global phase is. Nielsen and Chuang book does this a bit more gracefully, starting with the definition (saying that $e^{i\\theta}|\\psi\\rangle$ is the same as $|\\psi\\rangle$ up to a global phase factor $e^{i\\theta}$, where $\\theta$ is a global phase) and then proceed to postulate that global phase is not observable. I'd suggest following the same flow here.